### PR TITLE
Refs #27702 - creates export button component

### PR DESCRIPTION
**Pull request by MariaAga**
a sub task from https://github.com/theforeman/foreman/pull/6943
creates a basic export component and adds a helper to get the most common export url.

### CODE REVIEW

**Review by tbrisker — CHANGES_REQUESTED** [review]
Looks good! 
you can use subtasks in redmine to make it easy to break up a complex task into smaller chunks instead of using multiple `refs` commits - I've gone ahead and created https://projects.theforeman.org/issues/27702 for this one. Please update the commit message and rebase on develop to get the tests green, then we can merge this.

File: webpack/assets/javascripts/react_app/pages/common/PageLayout/components/ExportButton/ExportButton.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { testComponentSnapshotsWithFixtures } from 'react-redux-test-utils';
+
+import ExportButton from './ExportButton';
+
+const fixtures = {
+  'render with minimal props': { url: 'url' },
```
Comment (amirfefer) [inline webpack/assets/javascripts/react_app/pages/common/PageLayout/components/ExportButton/ExportButton.test.js]: what about having another test with `text` and `title` props?

**Review by amirfefer — CHANGES_REQUESTED** [review]
Thanks @MariaAga , looks good :+1:  just a minor thing :)

**Review by sharvit — CHANGES_REQUESTED** [review]
Thank you @MariaAga :+1: 

Can you please add a storybook for this component?
(AFAIK the rule is not to merge unused components to the code :disappointed: )

File: webpack/assets/javascripts/react_app/pages/common/PageLayout/components/ExportButton/ExportButton.test.js (hunk 1 above)
Comment (MariaAga) [reply webpack/assets/javascripts/react_app/pages/common/PageLayout/components/ExportButton/ExportButton.test.js]: Added, thanks!

**Comment by MariaAga** [comment]
@tbrisker @sharvit  Added storybook and changed the commit message, thanks for the review

**Review by amirfefer — DISMISSED** [review]
LGTM, thanks @MariaAga  :+1:

File: webpack/assets/javascripts/react_app/pages/common/PageLayout/components/ExportButton/ExportButton.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ExportButton.propTypes = {
+  url: PropTypes.string.isRequired,
+  title: PropTypes.string,
+  text: PropTypes.string,
+};
+
+ExportButton.defaultProps = {
+  title: __('Export to CSV'),
+  text: __('Export'),
+};
```
Comment (tbrisker) [inline webpack/assets/javascripts/react_app/pages/common/PageLayout/components/ExportButton/ExportButton.js:23]: Should the url default to exportUrl?

Comment (MariaAga) [reply webpack/assets/javascripts/react_app/pages/common/PageLayout/components/ExportButton/ExportButton.js:23]: that makes sense, thanks

**Comment by MariaAga** [comment]
test failure seems unrelated

**Review by tbrisker — APPROVED** [review]
Thanks @MariaAga !

**Pull request merged**